packages/pci-sdk-web/src/index.ts

```diff
@@ -179,9 +186,10 @@ function _checkInitOptions(initOptions: InitOptions) {
 function _initIframe(
 	authOptions: IAuthOptions,
 	pciElement: HTMLElement | null = document.getElementById('apto-pci-sdk'),
+	networkLogo?: INetworkLogo,
 	size?: Size,
 	values?: Values,
-	theme = '1',
+	theme?: string,
```

Comment:
Theme is not any string is an enum `light` or `dark` .

I found a bug yesterday because this should be initialised to one of those values not `1`